modules/packages/Sort.chpl

```diff
@@ -331,18 +419,35 @@ proc radixSortOk(Data: [?Dom] ?eltType, comparator) param {
 }
 
 /*
-   General purpose sorting interface.
 
-   :arg Data: The array to be sorted
-   :type Data: [] `eltType`
-   :arg comparator: :ref:`Comparator <comparators>` record that defines how the
-      data is sorted.
+Sort the elements in an array.
```

Comment:
So, do we want this to be a blackbox sort function or do we want to document what conditions will lead to what kind of sort function?

Right now, we document the condition in which parallel radix sort is used, otherwise, it is unknown to the user.